docs/learn/encyclopedia/storage/state-archival.mdx

```diff
@@ -63,12 +63,18 @@ env.storage().persistent().get(&EXAMPLE_KEY); // Returns Ok(1)
 env.storage().temporary().get(&EXAMPLE_KEY); // Returns Ok(2)
 ```
 
-All `Instance` storage is stored in a single contract instance `LedgerEntry` and shares a single TTL. This means that one call to `Env.storage().instance().extend_ttl()` will extend the TTL of all `Instance` entries, as well as the contract instance itself. For `Temporary` and `Persistent` storage, each entry has its own TTL and must be extended individually. The interface is slightly different and takes the key of the entry being extended as well as the TTL extension value.
+All `Instance` storage is stored in a single contract instance `LedgerEntry` and shares a single TTL. This means that one call to `env.storage().instance().extend_ttl()` will extend the TTL of all `Instance` entries, as well as the contract instance itself. `env.storage().instance().extend_ttl()` will also extend the contract code entry as well, which we'll expand on in the next section. For `Temporary` and `Persistent` storage, each entry has its own TTL and must be extended individually. The interface is slightly different and takes the key of the entry being extended as well as the TTL extension value.
```

Comment:
```suggestion
All `Instance` storage is stored in a single contract instance `LedgerEntry` and shares a single TTL. This means that one call to `env.storage().instance().extend_ttl()` will extend the TTL of all `Instance` entries, as well as the contract instance itself. It will also extend the contract code entry as well, which we'll expand on in the next section. For `Temporary` and `Persistent` storage, each entry has its own TTL and must be extended individually. The interface is slightly different and takes the key of the entry being extended as well as the TTL extension value.
```